org.audacious_media_player.Audacious.yaml

```diff
@@ -0,0 +1,160 @@
+# WIP
+id: org.audacious_media_player.Audacious
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: audacious
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-music
+  - --own-name=org.atheme.audacious
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+rename-desktop-file: audacious.desktop
+rename-icon: audacious
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/include'
+  - '/lib/girepository-1.0'
+  - '/lib/pkgconfig'
+  - '/share/doc'
+  - '/share/gir-1.0'
+  - '/share/gtk-doc'
+  - '/share/info'
+  - '/share/man'
+modules:
```

Comment:
Please follow these guidelines for best readability: https://github.com/flathub/flathub/wiki/YAML-Style-Guide